examples/guestbook-go/README.md

```diff
@@ -59,14 +59,14 @@ This example assumes that you have a working cluster. See the [Getting Started G
 
 Use the `examples/guestbook-go/redis-master-controller.json` file to create a [replication controller](../../docs/user-guide/replication-controller.md) and Redis master [pod](../../docs/user-guide/pods.md). The pod runs a Redis key-value server in a container. Using a replication controller is the preferred way to launch long-running pods, even for 1 replica, so that the pod benefits from the self-healing mechanism in Kubernetes (keeps the pods alive).
 
-<nop>1. Use the [redis-master-controller.json](redis-master-controller.json) file to create the Redis master replication controller in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
+1. Use the [redis-master-controller.json](redis-master-controller.json) file to create the Redis master replication controller in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
 
     ```console
     $ kubectl create -f examples/guestbook-go/redis-master-controller.json
     replicationcontrollers/redis-master
     ```
 
-<nop>2. To verify that the redis-master-controller is up, list all the replication controllers in the cluster with the `kubectl get rc` command:
+2. To verify that the redis-master controller is up, list all the replication controllers in the cluster(in `default` namespace) with the `kubectl get rc` command:
 
```

Comment:
How about this:
(I put `you created` in the sentence which has a `list all`, and an enhanced comment only in the *first* sentence)

```
list all the replication controllers you created in the cluster with the `kubectl get rc` command(if you don't specify a `--namespace` flag, the `default` namespace will be used. The same below):
...

list all the pods you created in cluster with the `kubectl get pods` command:
...

list all the services you created in the cluster with the `kubectl get services` command:
```